backend/apis/clients/get-client.api.js

```diff
@@ -9,7 +9,13 @@ app.get("/api/clients/:id", (req, res, next) => {
       return databaseError(req, res, err, connection);
```

Comment:
Can the view clients path be the same..(/api/clients) We had originally planned to use encoded url params to query/search clients or get all clients if no params are submitted..

Reply:
this is a different url, because it has a specific client ID in it

/api/clients - list of clients
/api/clients/:id - one specific client